src/app/core/http-services/item-navigation.service.ts

```diff
@@ -126,7 +126,7 @@ function createNavMenuItem(raw: {
   return {
     id: raw.id,
     title: raw.string.title ?? '',
-    hasChildren: raw.has_visible_children,
+    hasChildren: raw.has_visible_children && raw.permissions.can_view !== 'none' && raw.permissions.can_view !== 'info',
```

Comment:
Idea for improve - I prefer to use:

!['none', 'info'].includes(raw.permissions.can_view);

Reply:
```suggestion
    hasChildren: raw.has_visible_children && ![ 'none', 'info' ].includes(raw.permissions.can_view),
```